sensors/SensorCommon.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC, abstractmethod
+import numpy as np
+from gym import spaces
+
+
+def Dist2Circ(x1, y1, x2,  y2, r):
+    dx = x2 - x1
+    dy = y2 - y1
+    Dist2Center = (dx ** 2 + dy ** 2) ** (1 / 2)
+    if Dist2Center < r:
+        return np.zeros(2)
+    dx = dx * (Dist2Center - r) / Dist2Center
+    dy = dy * (Dist2Center - r) / Dist2Center
+    return dx, dy
```

Comment:
Make variables and functions lowercase to not confuse them with classes.
Would it be possible to pass [x1, y1] as array directly and not have the explicit splitting between the two variables?
That would make it easier later to extend to 3D if we want to and it makes the code shorter and cleaner.

Reply:
will do. good point, I will change the function to take arrays.